CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+## Change Log
+
+### 1.0.0 -> 1.1.0
```

Comment:
Nice that there's a changelog now!

Nits:

- For the changelog's I've seen, the heading is for the new version only: so `1.1.0` rather than `1.0.0 -> 1.1.0`
- You can probably promote all headers (i.e., drop one of the `#` characters, so that "Change log" is at level 1, etc.

Reply:
+1 to @chalin !
```suggestion
### 1.1.0
```